environments/fetcher/cmd/main.go

```diff
@@ -20,9 +20,12 @@ import (
 // Usage: fetcher <shared volume path>
 func main() {
 	flag.Usage = fetcherUsage
+	specializeOnStart := flag.Bool("specialize-on-startup", false, "Flag to activate specialize process at pod starup")
 	fetchPayload := flag.String("fetch-request", "", "JSON Payload for fetch request")
 	loadPayload := flag.String("load-request", "", "JSON payload for Load request")
-	specializeOnStart := flag.Bool("specialize-on-startup", false, "Flag to activate specialize process at pod starup")
+	secretDir := flag.String("secret-dir", "", "Path to shared secrets directory")
```

Comment:
One question here. Choosing the path for mounting secrets and configMaps in the fetcher means that all functions are forced to use the same path for their secrets and configs. right?
There may be a few users who would like to choose a specific path for their secrets and configMaps during `fn create --secret <name> --secretPath </foo/bar>`. 
Would it be a good idea to not make these volumes at the startup of fetcher, instead, create them during `specializePod` function? 
I do understand that this approach may slow down the start of function a bit, but, i think it might be worth a tradeoff of speed for flexibility here. 
This is just a though. I'm willing to hear what you all have to say.